db/seeds.rb

```diff
@@ -45,35 +53,35 @@ def base_seeds!
       )
     sdgfw.save!
 
-    # Set up taxonomies
-    # Global taxonomy
+    # Set up taxonomies ########################################################
+    # 1. Global taxonomy
     body = FactoryGirl.create(
```

Comment:
I am surprised to see `FactoryGirl` given we use `factory_bot_rails` which should be using `FactoryBot`

Reply:
only replaced it in 2023 when I brought in updates from marine-defrag
https://github.com/impactoss/impactoss-server/commit/cbcb8bf64da89573392e15f39533c3c7d3a0b799